CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+## Reporting Issues
+
+If you have found what you think is a bug, please [file an issue](https://github.com/pmndrs/jotai/issues/new).
+
+## Suggesting new features
+
+If you are here to suggest a feature, first create an issue if it does not already exist. From there, we will discuss use-cases for the feature and then finally discuss how it could be implemented.
+
+## Development
+
+If you have been assigned to fix an issue or develop a new feature, please follow these steps to get started:
```

Comment:
Have you confirmed these steps work for you? I'm not very confident with them.

Reply:
no, it's not quite right.
especially the auto-build part and example folder.

Reply:
Would you modify them or remove the section?